content/docs/sidebar.json

```diff
@@ -100,8 +100,11 @@
         "slug": "how-to",
         "source": false,
         "children": [
-          "add-output-to-stage",
           "undo-adding-data",
+        {
+          "label": "Add Tracked Data to a Stage",
+          "slug": "add-deps-or-outs-to-a-stage"
+        },
           "update-tracked-files"
```

Comment:
Also, there's a formatting error here according to https://app.circleci.com/pipelines/github/iterative/dvc.org/6615/workflows/b705d2d8-9861-41a9-8a2b-e18222cd66b1/jobs/6671

Probably indentation?

Reply:
Please remember to follow https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown

Reply:
Done.